test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -1140,6 +1171,92 @@ class TestDapp {
     console.log(`Check testdapp is connected to network ${networkId}`);
     await this.driver.waitForSelector(this.connectedNetwork(networkId));
   }
+
+  /**
+   * Check the number of dapps connected
+   *
+   * @param numberOfDapps - The number of dapps connected
+   */
+  async checkNumberOfDappsConnected(numberOfDapps: string) {
+    await this.driver.waitForSelector(this.dappNumberConnected(numberOfDapps));
+  }
```

Comment:
if I'm not mistaken, this refers to the MM Dialog window, so we shouldn't add this method here, rather in the correct page object (t's possible it already exists)